docs/installation.rst

```diff
@@ -78,20 +78,40 @@ Standard Installation
 
 3. Save ``generated_values.yaml``, somewhere safe. This is your Helm ``values.yaml`` file.
 
-4. Download the Helm chart and Install Robusta using Helm. On some clusters this can take a while [#f2]_, so don't panic if it appears stuck:
+.. admonition:: Multiple clusters
+    :class: important
+
+    Use the same ``generated_values.yaml`` for all your clusters (dev, prod, etc..). There's no need to run gen-config again.
+
+Standard Installation
+------------------------------
+
+1. Download the Helm chart:
```

Comment:
I think download is the wrong wording.
You can download a local copy of the help chart, buy running `helm pull --untrar ...`

But the default way to install is just adding our helm repo, to the helm client, and work from there.

I would name this: Add Robusta's chart reppository
 (or something)